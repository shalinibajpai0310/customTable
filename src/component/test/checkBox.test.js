import React from 'react';
import { shallow } from 'enzyme';
import CheckBox from '../checkBox';

describe('<CheckBox />', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<CheckBox />);
    expect(wrapper).toMatchSnapshot();
  });
});