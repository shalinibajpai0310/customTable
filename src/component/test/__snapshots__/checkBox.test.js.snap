// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CheckBox /> renders without crashing 1`] = `ShallowWrapper {}`;
